packages/core/src/internal/coreEffectRunners/select.js

```diff
@@ -0,0 +1,17 @@
+import * as effectTypes from '../effectTypes'
+import { is } from '../utils'
+
+export default function select(env, payload, cb) {
```

Comment:
should we unify signatures across effects? sometimes we destructure it, sometimes use as is - it not obvious that all of them have the same signature

Reply:
agree